### DISCUSSION TITLE
X-osint has been updated to Version 2.3

### DISCUSSION THREAD

**Discussion by TermuxHackz** [TermuxHackz/X-osint#56] [Announcements]
Hello there folks, @LuiDavinci , @TermuxHackz , @spider863644 , @spider863644 , @Cr7op , @rdxbubble , X-osint has been updated to Version 2.3 bringing to you more OSINT tools and of course bug fixes and improvements, Kindly update X-osint manually or automatic, read [here](https://github.com/TermuxHackz/X-osint#how-to-update-manually-for-any-version) on how to update manually and Read [here](https://github.com/TermuxHackz/X-osint#how-to-update-automatically-if-your-using-version-21-of-x-osint-and-above) on how to update Automatically.

Please ensure you update X-osint as it's important. 
Read [here](https://github.com/TermuxHackz/X-osint#changelogs-for-23) for the Changelogs for version 2.3 and check out whats new

**Comment by spider863644** [comment]
Thanks

**Comment by spider863644** [comment]
Thanks
